WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
 		
 		allow = {
-			hidden_tooltip = {
+			OR = {
+				religion = bosporan_faith
```

Comment:
There shouldn't BE limits on this DE JURE title at all, IMO, but in this case, it makes no sense. Right now, if a Bosporan captures all of Tauricia (and thus most of his holy sites), he cannot create a Kingdom and will be doomed to Gavelkind. That makes no sense.